README.md

```diff
@@ -213,7 +212,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 - [Lobsters](https://lobste.rs) - Run your own link aggregation site. ([Source Code](https://github.com/jcs/lobsters)) `BSD` `Ruby`
 - [No Fuss Bookmarks](http://nofussbm.herokuapp.com/signup.html) - Very simple software and service to store bookmarks especially designed for hackers (that don't need fancy interfaces, but nice API). ([Source Code](https://github.com/mapio/nofussbm)) `GPL-3.0` `Python`
 - [Pinry](http://getpinry.com/) - The tiling image board system for people who want to save, tag, and share images, videos, and webpages. ([Source Code](https://github.com/pinry/pinry)) `BSD-2-Clause` `Python`
-- [saveto](https://save.duyet.net/) - Open source, home for the best links on the web. ([Source Code](https://github.com/saveto-co/saveto)) `MIT` `Nodejs`
+- [saveto](https://github.com/saveto-co/saveto) - Open source, home for the best links on the web. `MIT` `Nodejs`
```

Comment:
Project looks abandoned since 2017 and [security-related pull requests](https://github.com/saveto-co/saveto/pulls) are piling up. I think it is safe to remove it completely. @duyetdev is saveto still maintained?

Reply:
It is better to remove it then